orchestration/src/main/java/com/sap/ai/sdk/orchestration/spring/OrchestrationSpringChatResponse.java

```diff
@@ -20,26 +20,25 @@
 /** Response from the orchestration service in a Spring AI {@link ChatResponse}. */
 @Value
 @EqualsAndHashCode(callSuper = true)
-public class OrchestrationChatResponse extends ChatResponse {
+public class OrchestrationSpringChatResponse extends ChatResponse {
```

Comment:
(Major)

You are breaking public API.
We can only allow that when PO gave approval.

Reply:
Sorry made a mistake